### PR TITLE
Fill out more docs

### PR DESCRIPTION
**Pull request by clburlison**
The Clustering Modes could use a review for accuracy. It's likely close enough but might not be 100% technically correct.

### CODE REVIEW

File: docs/pages/client/map.mdx [hunk 1]
```diff
@@ -41,12 +41,20 @@ import { Callout } from 'nextra-theme-docs'
...
 
+Most these options are up for interpretation on which results are best for your area. Some areas and MITM clients might prefer one result over another so it is recommended you do some testing. If you want a good starting point try **Brute Force**.
+
 - Min Points
   - Minimum number of points that a cluster should have
-- Fast
-  - Uses an exponentially faster algorithm but returns worse results
-- 🚧 Only unique 🚧
-  - Counts min points by unique points instead of all points in a Cluster
+- Cluster Mode
```
Comment (TurtIeSocks) [inline docs/pages/client/map.mdx:48]: I should probably just combine cluster and route modes with a "Order By" drop down that has a "none" option... but for now, this is a good explanation.

File: docs/pages/client/admin.mdx [hunk 2]
```diff
@@ -39,6 +39,17 @@ import { Callout } from 'nextra-theme-docs'
 
 - Create, edit, and delete your various tileservers
 - Tileservers are used for selecting different map tiles from the Kōji client
+- Currently only tested with: [123FLO321/SwiftTileserverCache](https://github.com/123FLO321/SwiftTileserverCache)
```
Comment (TurtIeSocks) [inline docs/pages/client/admin.mdx:42]: It also works with the standard remote ones. OSM, Satellite, Voyager, etc. Not sure if it's worth mentioning alongside the custom examples?

Comment (clburlison) [reply docs/pages/client/admin.mdx]: Agreed. I'm not including all common servers in this example but at least include the Default that Koji ships with.

**Review by TurtIeSocks — APPROVED** [review]
Thank you!

**Pull request merged**